includes/enrolment/class-sensei-enrolment-course-calculation-job.php

```diff
@@ -153,4 +188,144 @@ private function get_query_args( $course_enrolment ) {
 
 		return $user_args;
 	}
+
+	/**
+	 * Modify user query to add the user ID check.
+	 *
+	 * @access private
+	 *
+	 * @param WP_User_Query $user_query User query to modify.
+	 */
+	public function modify_user_query_add_user_id( WP_User_Query $user_query ) {
+		global $wpdb;
+
+		$user_query->query_where .= $wpdb->prepare( ' AND ID>%d', $this->get_last_user_id() );
```

Comment:
Do we need to order by id ID for this to work? I think that WP_User_Query sorts by `user_login` by default, unless I miss something.

Reply:
Ahh, shoot... yup! I had it in there but think I accidentally removed it. Adding it back in...b883850566c7f7f05c3ebd15280c874bfc34dc43